src/pages/s/[[...all]].tsx

```diff
@@ -99,7 +101,7 @@ export const getServerSideProps: GetServerSideProps = async function ({
 
   return {
     props: {
-      resultsState: {rawResults},
+      resultsState: JSON.parse(JSON.stringify(resultsState)),
```

Comment:
I changed this a bit so that instead of destructuring the resultsState that algolia feeds us we pass along the full object and do the destructuring in a separate step.

Reply:
Nice, encountered the same issue a while back, this solved it for me too!